parachain/pallets/ethereum-beacon-light-client/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "snowbridge-ethereum-beacon-light-client"
+description = "Snowbridge Beacon Light Client Pallet"
+version = "0.0.1"
+edition = "2021"
+authors = ["Snowfork <contact@snowfork.com>"]
+repository = "https://github.com/Snowfork/snowbridge"
+license = "PENDING/TBC"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.130", optional = true }
+codec = { version = "2.2.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+milagro_bls = { git = "https://github.com/sigp/milagro_bls", tag = "v1.4.2", default-features = false }
+ssz_rs = { git = "https://github.com/Snowfork/ssz_rs" }
+ssz_rs_derive = { git = "https://github.com/Snowfork/ssz_rs" }
+
+frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false, optional = true }
```

Comment:
In the main branch we upgraded all Substrate/Cumulus/Polkadot dependencies to `polkadot-v0.9.17`. And you merged those changes back into your branch. So should make sure you update this pallet too. :)